controllers/contacts-controller.js

```diff
@@ -1,15 +1,15 @@
-const contactsService = require('../models/contacts');
+const {Contact} = require('../models/contact');
 const { HttpError } = require('../helpers');
 const { ctrlWrapper } = require('../decorators');
 
 const getAllContacts = async (req, res) => {
-	const result = await contactsService.listContacts();
+	const result = await Contact.find();
```

Comment:
```suggestion
	const result = await Contact.find();
```
сервіси можна використовувати як раніше. Тільки переписати логіку, щоб сервіс робив запити в бд

Reply:
та повертав об'єкт